src/Language/PureScript/Docs/Types.hs

```diff
@@ -307,9 +307,12 @@ asDeclaration =
               <*> key "fixity" (perhaps asFixity)
               <*> key "info" asDeclarationInfo
 
-asFixity :: Parse PackageError P.Fixity
-asFixity = P.Fixity <$> key "associativity" asAssociativity
-                    <*> key "precedence" asIntegral
+asFixity :: Parse PackageError (P.Fixity, Maybe String)
+asFixity = do
+  fixity <- P.Fixity <$> key "associativity" asAssociativity
+                     <*> key "precedence" asIntegral
+  alias <- key "alias" (perhaps asString)
```

Comment:
Apparently not, `perhaps` is kind of like purescript's `Data.Foreign.Null.readNull`: https://hackage.haskell.org/package/aeson-better-errors-0.9.0/docs/Data-Aeson-BetterErrors.html#v:perhaps
But there is also `keyMay`, which would give you `Nothing` if the property wasn't there, so as far as I can tell, if we used that, we would probably be able to keep accepting uploads from people on 0.7.x. I'm not really sure about what to do here; it's hard to predict whether that would be worth doing.

By the way, does this line up with the `ToJSON` instance? You can run the test in `psc-publish/test` to do a quick to-JSON-and-back test. (I should make that into an actual test.) I think this might be using Aeson's `ToJSON (,)` instance at the moment, which might not work with this parser.